.github/workflows/bors.toml

```diff
@@ -0,0 +1,4 @@
+status = [ "ci" ]
+
+timeout_sec = 7200
+delete_merged_branches = true
```

Comment:
Why is it in the workflows folder? Is it a Github Bot configuration file?

Reply:
yes, it's will go to Bot configuration file before the PR merge.